source/World.hpp

```diff
@@ -67,6 +67,38 @@ namespace GTA
 		SmallExplosion7 = 33,
 		Explosion6 = 34
 	};
+	public enum class MarkerTypes
+	{
+		UpsideDownCone = 0,
+		VerticalCylinder = 1,
+		Thick_Chevron_Up = 2,
```

Comment:
Going with the naming convention, could you change all the names to the CamelCase-style? Like this to `ThickChevronUp`. :)